docs/guide/types/box.md

```diff
@@ -118,14 +118,20 @@ All of these options can be [Scriptable](../options#scriptable-options)
 | `font` | [`Font`](../options#font) | `{ weight: 'bold' }` | Label font
 | `height` | `number`\|`string` | `undefined` | Overrides the height of the image or canvas element. Could be set in pixel by a number, or in percentage of current height of image or canvas element by a string. If undefined, uses the height of the image or canvas element. It is used only when the content is an image or canvas element.
 | `padding` | [`Padding`](../options#padding) | `6` | The padding to add around the text label.
-| [`position`](#position) | `string`\|`{x: string, y: string}` | `'center'` | Anchor position of label in the box.
+| [`position`](#position) | `string`\|`{x: string, y: string}`\|`number` | `'center'` | Anchor position of label in the box.
```

Comment:
I think the type should be `string`\|`{x: string\|number, y: string\|number}`\|`number` (number is allowed for the x/y object too)

Reply:
yes!!

Reply:
done